constant/web3/supportedChains.ts

```diff
@@ -9,3 +9,16 @@ export const supportedChains = {
   43114: "Avalanche",
   42161: "Arbitrum",
 };
+
+export function getOracleProperName(oracleType:string):string{
```

Comment:
nit: spacing and redundant return type

Reply:
thats wierd, why is the linter not picking that up